settings/development.py

```diff
@@ -77,3 +77,9 @@
 
 # This will expose all browsable api urls. For dev the default value is true
 API_DEBUG = env.bool('API_DEBUG', default=True)
+
+# MEDIA CONFIGURATION
+# ------------------------------------------------------------------------------
+
+# Media configuration to support deployment of media files while is debug=True or development.
+MEDIA_URL = env("MEDIA_URL", default="media/")
```

Comment:
This is not needed. The `MEDIA_URL` is set to `.media` directory already.

Reply:
`MEDIA_ROOT` is set to `.media` to signify the place from where django should get the media files. `MEDIA_URL` is set for the deployment and has to be set to the path from where media files can be picked up while the server is running. [source](https://docs.djangoproject.com/en/2.1/ref/settings/#std:setting-MEDIA_URL)

Reply:
This is already in the `settings/common.py`. Please remove this change from here.

Reply:
The setting currently in `common.py` is not fit for development. I've already created a [pull](https://github.com/Fueled/django-init/pull/324) in Django-init to resolve this.